luatest/hooks.lua

```diff
@@ -4,6 +4,30 @@ local comparator = require('luatest.comparator')
 
 local export = {}
 
+local preloaded_hooks = {
+    before_suite = {},
+    after_suite = {},
+
+    before_all = {},
+    after_all = {},
+}
+
+function export.before_suite_preloaded(fn)
```

Comment:
As I understood, the `luatest.hooks` module is now public and can be used in tests/testing helpers.

It means that all its functions (except unes that start from `_`) are public or at least it looks like they're public.

However, other existing functions are for internal use, as I see.

I think it would be good to differentiate private and public functions somehow.

Reply:
Not here. Create issue, please